wbooks/src/app/screens/dashboard/screens/Home/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import BookList from './components/BookList';
+import SearchBar from './components/SearchBar';
+import BookServices from '../../../../services/BookServices';
+import './styles.css';
+
+
+
+class Home extends React.Component {
+
+  state = {
+    searchValue: "",
+    libros: BookServices.books
+  }
+
+  handleSubmit = (value) => {
+
+    this.setState({searchValue: value});
+
+    this.setState({ libros: BookServices.books.filter((book) => {
+      return ( book.title.indexOf(this.state.searchValue) !== -1 || book.author.indexOf(this.state.searchValue) !== -1 )
```

Comment:
Extract this boolean condition into a function:

```
shouldIncludeBook(book, field, filter) {
  return book[field].indexOf(filter) !== -1;
}
...
// In this setState
this.setState({ libros: BookServices.books.filter(book => this.shouldIncludeBook(book, 'title', value) || ... });
```
If you like to, you can customize this to include all cases specifying a list of fields.